apps/core/src/utils/transaction/getGroupByOwner.ts

```diff
@@ -36,13 +36,13 @@ export function getGroupByOwner(objectSummaryChanges: ObjectSummaryChange[]) {
 
             let key;
             let locationIdType;
-            if ('AddressOwner' in owner) {
+            if (owner !== 'Immutable' && 'AddressOwner' in owner) {
```

Comment:
I don't think it's great to still have the misleading type around, this is bound to cause another surprise issue down the road 😄. I was actually a fan of the other change you made to default `owner` to `null`/`undefined`

Reply:
lol let me change it back